drivers/sensor/tcs3472x/tcs3472x.c

```diff
@@ -0,0 +1,216 @@
+/* tcs3472x.c - Driver for AMS TCS3472X color sensor */
+
+/*
+ * Copyright (c) 2023, Thomas Kiss <thokis@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/init.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/__assert.h>
+
+#include <zephyr/logging/log.h>
+
+#include "tcs3472x.h"
+#include "zephyr/drivers/i2c.h"
+#include <stdint.h>
+#include <stdlib.h>
```

Comment:
I think you can remove gpio, stdint, and stdlib. Please also change i2c to a system include.